src/pages/Orders.js

```diff
@@ -1,44 +1,70 @@
-import styled from 'styled-components';
+import styled, {css} from 'styled-components';
 
 import Header from '../components/Header/Header.js';
 import Navigation from '../components/Navigation/Navigation';
+import useStoreLogin from '../hooks/useStore_login';
```

Comment:
If you change the name of the useStore according to my suggestion, please keep in mind to change it here too.🙂